### ISSUE TITLE
Mouse rotation speed range

### ISSUE THREAD

**Issue by Garux** [TTimo/GtkRadiant#53]
Setting rotation speed to 1 still results in too high sensitivity with 5k+ dpi mouse.
Tried to set not integer value, radiant doesn't accept one.

**Comment by Pan7** [comment]
Can you test this in the latest version of radiant?

**Comment by Garux** [comment]
No, latest version of radiant doesn't support my OS Windows version :3

**Comment by ensiform** [comment]
Upgrade from Windows XP.

**Comment by Garux** [comment]
kek
Need to upgrade pc first to make OS upgrade somewhat beneficial.
There are no real drawbacks from supporting XP in most softwares; Most of those, who do not, simply 'do not care'.
As for dropping support in this radiant branch, it is fairly ridiculous motivator to upgrade, since it's far not leader in stability/functionality/performance/codebase development pespective among existing radiants; Bestest feature of this one is version number :)
Also there are fresh editors, supporting idtech3 games, which care to be executable on windows 5.x version

**Comment by Garux** [comment]
@Pan7 : i'v tried GtkRadiant-1.6.5-20170303 on supported os and no, your commit about mouse sensitivity does nothing (had you even take a look on what was saving and what was loading? try); Also setting <1 via slider isn't available
One more comedy is running there about settings:
1st start saves them to
\q3.game\local.pref
2nd
\installs\Q3Pack\game\local.pref
